<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExportPages</class>
 <widget class="QDialog" name="DialogExportPages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Page</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBoxExportPages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>277</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxPrintOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Export</string>
   </property>
   <widget class="QLabel" name="labelCurrentPage">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCurrentPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPageRange">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>90</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter page numbers and/or page ranges. For example, 1,3,5,7-9</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPageRange">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Page Range</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAllPages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>All Pages</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAllPages">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>labelCurrentPage</zorder>
   <zorder>radioButtonCurrentPage</zorder>
   <zorder>labelAllPages</zorder>
   <zorder>radioButtonAllPages</zorder>
   <zorder>lineEditPageRange</zorder>
   <zorder>radioButtonPageRange</zorder>
   <zorder>labelAllPages</zorder>
   <zorder>radioButtonAllPages</zorder>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxExportCache">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>151</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Cache Management</string>
   </property>
   <widget class="QCheckBox" name="checkBoxResetCache">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Check to clear all caches before printing/export</string>
    </property>
    <property name="text">
     <string>Reset all caches before content printing/export.</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMixedPageSizes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>208</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Mixed Size and/or Orientation Warning</string>
   </property>
   <widget class="QCheckBox" name="checkBoxIgnoreMixedPageSizes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ignore mixed page size and/or orientation message when previewing pages.</string>
    </property>
    <property name="text">
     <string>Do not show mixed size and/or orientation message</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxExportPages</sender>
   <signal>accepted()</signal>
   <receiver>DialogExportPages</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxExportPages</sender>
   <signal>rejected()</signal>
   <receiver>DialogExportPages</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
